consensus/src/dag/bootstrap.rs

```diff
@@ -472,21 +543,30 @@ impl DagBootstrapper {
         mut dag_rpc_rx: Receiver<Author, IncomingDAGRequest>,
         mut shutdown_rx: oneshot::Receiver<oneshot::Sender<()>>,
     ) {
+        info!(
+            LogSchema::new(LogEvent::Start),
```

Comment:
I found that Start is too ambiguous to search in logs, probably rename to EpochStart